docs/release/release.md

```diff
@@ -0,0 +1,34 @@
+## Preparing the release
+
+1. Create a release branch in the form of `release/[version]`
```

Comment:
instead on [version] i think DD-MM-YYYY-HH:MM or something would be better. The version is generated with git-version. This might trick you

Reply:
Done